client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -1,452 +1,122 @@
-                    disabled
-                  />
-                }
-                labelWidth={LABEL_WIDTH}
-                label={t('label.suggested-quantity')}
-                sx={{ marginBottom: 1 }}
-              />
-              {isProgram && useConsumptionData && (
-                <InputWithLabelRow
-                  Input={
-                    <ReasonOptionsSearchInput
-                      value={draft?.reason}
-                      onChange={value => {
-                        update({ reason: value });
-                      }}
-                      width={200}
-                      type={ReasonOptionNodeType.RequisitionLineVariance}
-                      isDisabled={
-                        draft?.requestedQuantity === draft?.suggestedQuantity ||
-                        isSent
-                      }
-                      onBlur={save}
-                    />
-                  }
-                  labelWidth={'66px'}
-                  label={t('label.reason')}
-                  sx={{ marginBottom: 1 }}
-                />
-              )}
-              <InputWithLabelRow
-                Input={
-                  <TextArea
-                    value={draft?.comment ?? ''}
-                    onChange={e => update({ comment: e.target.value })}
-                    slotProps={{
-                      input: {
-                        sx: {
-                          backgroundColor: theme =>
-                            theme.palette.background.menu,
-                        },
-                      },
-                    }}
-                    onBlur={save}
-                    disabled={isSent}
-                  />
-                }
-                sx={{ width: 275 }}
-                labelWidth={LABEL_WIDTH}
-                label={t('label.comment')}
-              />
-            </Box>
-          </>
-        )}
-      </Box>
+  useEffect(() => {
+    if (data) setCurrentItem(data);
+  }, [data, setCurrentItem]);
 
+  return (
+    <Box display="flex" flexDirection="column" padding={2} gap={1}>
+      <AccordionPanelSection
+        key={`${key}_item_search`}
+        title={t('label.item', { count: 1 })}
+        closedSummary={draft?.itemName}
+        defaultExpanded={!disabled}
+      >
+        <StockItemSearchInput
+          currentItemId={draft?.itemId}
+          onChange={handleChange}
+          disabled={disabled}
+          extraFilter={item => !lines.some(line => line.item.id === item.id)}
+        />
+      </AccordionPanelSection>
+      <AccordionPanelSection
+        key={`${key}_order`}
+        title={t('title.order')}
+        defaultExpanded={!isNew && !disabled}
+      >
+        <Order
+          disabled={disabled}
+          isPacksEnabled={isPacksEnabled}
+          draft={draft}
+          update={update}
+        />
+      </AccordionPanelSection>
+      <AccordionPanelSection key={`${key}_details`} title={t('label.details')}>
```

Comment:
Ok, couple last thoughts:

Maybe default expanded for Details section should be same as Order - when adding item, it should be collapsed when no item selected, then open after selected? "1 month of stock" doesn't make much sense here:

![Screenshot 2025-05-07 at 11 19 49 AM](https://github.com/user-attachments/assets/b40b746f-cb53-47de-b09e-3391f8cd6dd4)

Speaking of Order section, it's not very big, I think maybe keep open for "sent"/disabled orders too? Or, show quantity in the `closedSummary`... either way, I want to see how many I ordered!

![Screenshot 2025-05-07 at 11 22 27 AM](https://github.com/user-attachments/assets/9a2f1f5a-5766-449f-a862-ebb30e538293)

Reply:
Thanks @lache-melvin, good points right there. Actually feels so much smoother in general. Please see change here: https://github.com/msupply-foundation/open-msupply/pull/7508/commits/b4a818fbdfd8af703016094e57ba9b8e3175a9e5